frontend/src/components/Map/components/ViewPointHD/AnnotationItem.tsx

```diff
@@ -1,20 +1,46 @@
 import L from 'leaflet';
+import SVG from 'react-inlinesvg';
 import { GeoJsonProperties, Geometry } from 'geojson';
-import { Circle, CircleMarker, Polygon, Polyline, Tooltip, useMap } from 'react-leaflet';
+import { Circle, CircleMarker, Marker, Polygon, Polyline, Tooltip, useMap } from 'react-leaflet';
+import Image from 'next/image';
+import { TrekMarker } from 'components/Map/Markers/TrekMarker';
+import { optimizeAndDefineColor } from 'stylesheet';
 
 type Props = {
   geometry: Geometry;
   properties: GeoJsonProperties;
   id: string;
 };
 
+const Icon = ({ pictogramUri }: { pictogramUri: string }) => {
+  if (!pictogramUri) {
+    return null;
+  }
+  return pictogramUri.endsWith('.svg') ? (
+    <SVG src={pictogramUri} className="size-6" preProcessor={optimizeAndDefineColor()} />
+  ) : (
+    <Image loading="lazy" src={pictogramUri} width={16} height={16} alt="" />
+  );
+};
+
 const MetaData = ({ properties }: { properties: GeoJsonProperties }) => {
   if (properties === null || !properties.name) {
     return null;
   }
+  const pictogramUri = properties?.category?.pictogramUri;
+
   return (
     <Tooltip>
-      <span className="text-base">{properties.name}</span>
+      <span className="flex flex-wrap items-center gap-2">
+        {Boolean(properties.category?.label) && (
+          <>
+            {Boolean(pictogramUri) && <Icon pictogramUri={pictogramUri} />}
```

Comment:
```suggestion
            <Icon pictogramUri={pictogramUri} />
```

(condition is useless as your component already render `null` if `pictogramUri` is falsy.)